.github/workflows/code-quality-pr.yaml

```diff
@@ -10,13 +10,10 @@ jobs:
     steps:
       - name: Checkout changes
         uses: actions/checkout@v1
-      - name: Install PHP
-        uses: shivammathur/setup-php@master
+      - name: Install PHP and PHP Code Sniffer
+        uses: shivammathur/setup-php@v1
         with:
-          php-version: 7.2
-      - name: Install Composer dependencies
-        run: composer install --no-interaction --no-progress --no-suggest
+          php-version: '7.3'
```

Comment:
What's with this change? Do our tests still support 7.2?

Reply:
Yeah, this is just for the Code Quality tests which run externally. It has no effect on the unit tests.